main_test.go

```diff
@@ -8,13 +8,13 @@ import (
 // GORM_BRANCH: master
 // TEST_DRIVERS: sqlite, mysql, postgres, sqlserver
 
-func TestGORM(t *testing.T) {
-	user := User{Name: "jinzhu"}
-
-	DB.Create(&user)
+type TableA struct {
+	Name string `gorm:"size:100;-:migration"`
```

Comment:
hello, look create table ddl.
```
CREATE TABLE `tableA` ()
```
It is illegal,if you add another field, for example :
```
type TableA struct {
	Name string `gorm:"size:100;-:migration"`
	UserId   int    `gorm:"column:user_id"`
}
```
It works correctly

Reply:
Yes, but the issue is the -:migration should only work for alter column, but in here, this column will not be created

Reply:
`-:migration` means only disable the migration, so this column will not be created. @MittWillson

Reply:
@longlihale but if this so, how different between `-` and `-:migration`

Reply:
Oh i see